### PR TITLE
add Chris de Almeida (CDA) to delegates.txt

### PR DESCRIPTION
**Pull request by ctcpip**

### CODE REVIEW

File: delegates.txt [hunk 1]
```diff
@@ -60,6 +60,7 @@ Caroline Cullen (CCU)
 Chengzhong Wu (CZW)
 Chip Morningstar (CM)
 Chris Anderson (CAN)
+Chris de Almeida (CMD)
```
Comment (robpalme) [inline delegates.txt]: Would you consider `CDA` or `CAA` to match others?

Comment (ctcpip) [reply delegates.txt:63]: would `CD` be ok?

Comment (ctcpip) [reply delegates.txt]: done

**Comment by ctcpip** [comment]
@robpalme good to merge?  thanks

**Review by robpalme — APPROVED** [review]

**Pull request merged**